objc/live-updates-pub.m

```diff
@@ -0,0 +1,2 @@
+ARTRealtimeChannel *channel = [ably.channels get:@"dogecoin:usd"];
+[channel publish:@"rate" data:@{ @"price": @1.01, @"time": @([[NSDate date] timeIntervalSinceReferenceDate]) }];
```

Comment:
Why is there `@` in front of `1.01` and wrapping `[[NSDate date] timeIntervalSinceReferenceDate]`?

Reply:
in objective-c `@` is literal for NSNumber object, it's the same as `[NSNumber numberWithFloat:1.01];`
more [HERE](https://clang.llvm.org/docs/ObjectiveCLiterals.html)

Reply:
Okay great, thanks